sql/parse/parse.go

```diff
@@ -158,6 +162,30 @@ func convertShow(s *sqlparser.Show) (sql.Node, error) {
 		return plan.NewShowTables(&sql.UnresolvedDatabase{}), nil
 	case sqlparser.KeywordString(sqlparser.DATABASES):
 		return plan.NewShowDatabases(), nil
+	case sqlparser.KeywordString(sqlparser.FIELDS), sqlparser.KeywordString(sqlparser.COLUMNS):
+		// TODO(erizocosmico): vitess parser does not support EXTENDED.
+		// TODO(erizocosmico): LIKE is not implemented, so we won't support it yet.
```

Comment:
I saw this PR #427 implementing the `LIKE` expression, just saying to don't forget change this when that PR get merged.